djangoplicity/media/options/videos.py

```diff
@@ -189,8 +189,20 @@ class VideoOptions( ArchiveOptions ):
             'resources': ( 'old_video', ),
             'icons': { 'old_video': 'movie' }
         } ),
+        (ugettext_noop(u'Giphy'), {
+            'resources': (video_giphy_url_link, ),
+            'icons': {'video_giphy_url_link': 'giphy'}
+        }),
     )
 
+    # TODO: For this implementation it is necessary to modify the views of the archive model
+    # extras = (
+    #     (ugettext_noop(u'Giphy'), {
+    #         'resources': (video_giphy_url_link),
+    #         'icons': {'video_giphy_url_link': 'giphy'}
+    #     }),
+    # )
+
```

Comment:
What does this mean? we should not keep commented code unless it's really important to keep it.

Reply:
Ok, This part of the code was removed